scripts/bansearch.pl

```diff
@@ -218,7 +227,21 @@ sub RPL_WHOREPLY {
 
         return if(!$running);
 
-        (undef, $user, $host, $nick, $account, $real) = split(/ /, $data, 6);
+	# 0   1 2   3       4               5    6 7  8
+	# bw2 * ~pi rpi1.my irc.example.net rpi1 H :0 pi
```

Comment:
where did you see this response?

I think it is wrong

$ownnick, $channel_or_star, $user, $host, $server, $nick, $here_or_gone, $hopcount, $real

it's H/G for away status, not account

also, you are missing the split limit (if real name contains spaces)